contracts/aggregator/mainnet/flashloan/main.sol

```diff
@@ -673,7 +673,7 @@ contract FlashAggregator is Setups {
      * @notice Function to transfer fee to the treasury. Will be called manually.
      * @param _tokens token addresses for transferring fee to treasury.
      */
-    function transferFeeToTreasury(address[] memory _tokens) public {
+    function transferFee(address[] memory _tokens, address _to) public onlyOwner {
```

Comment:
@KaymasJain Can we emit event from this function?

Reply:
Ooh! Yes. We can. I'll add that as well.